### ISSUE TITLE
msconvert: verbosity for error handling?

### ISSUE THREAD

**Issue by lindsaypino** [ProteoWizard/pwiz#1483]
Hello! I was wondering if there's any way to turn on some kind of "verbose" setting that would give more details about error handling while running MSconvert. Is such an option supported?

Thanks! ~Lindsay

**Comment by chambm** [comment]
Not really. If you're talking about running under WINE, then there are WINEDBG options that could help diagnose a problem there but they are very verbose and hard to interpret.
